gips/inventory/dbinv/api.py

```diff
@@ -214,7 +214,7 @@ def add_product(**values):
     return p # in case the user needs it
 
 
-def update_or_add_asset(driver, asset, tile, date, sensor, name):
+def update_or_add_asset(driver, asset, tile, date, sensor, name, status):
```

Comment:
Suggest adding a default of 'complete' to to the `status` arg for this fn and `update_or_add_product`.  That way you don't have to modify `gips/scripts/archive.py` nor fix this function's unit test(s?).